data/human/deep missions.txt

```diff
@@ -631,7 +631,7 @@ mission "Deep: Mystery Cubes 4"
 		event "assisted the deep" 7
 		payment 1000000
 		log `Helped to destroy the last of the Navy Cruisers stolen by Beelzebub, but he has escaped. The Navy is still searching, but is no longer asking for help from merchants.`
-		log "People" "Beelzebub" `Beelzebub, as many call them, is one of the most notorious pirates in Human space. They're responsible for committing countless acts of crimes against both Humanity and the Republic government and are still at large somewhere in the galaxy despite a major setback.`
+		log "People" "Beelzebub" `Beelzebub, as many call them, is one of the most notorious pirates in human space. They're responsible for committing countless acts of crimes against both humanity and the Republic government and are still at large somewhere in the galaxy despite a major setback.`
```

Comment:
apparently, `human` as adjective is not capitalized: https://english.stackexchange.com/questions/47324/why-we-capitalize-all-race-names-but-our-own

Reply:
That is an excellent explanation